frontend-react/src/shared/Card/Card.module.scss

```diff
@@ -26,3 +26,8 @@
         border-radius: 0;
     }
 }
+
+:global(.usa-card__container) {
```

Comment:
@jpandersen87 do you see any issues with me overriding these styles?

Reply:
no, but it should go into the uswds overrides sheet instead